code/game/objects/machinery/cryopod.dm

```diff
@@ -451,7 +451,7 @@
 	user.despawn(src)
 
 
-/obj/machinery/cryopod/proc/go_out()
+/obj/machinery/cryopod/proc/go_out(mob/source, gibbing)
```

Comment:
accidental change? this doesnt do anything far as i can tell

Reply:
Yep, looks like a mistake.